app/scripts/proactive/model/Job.js

```diff
@@ -270,6 +270,12 @@ define(
                     message: "<br><br>" + validationData.errorMessage
                   };
                   return err;
+                } else if (formValues.Model === "PA:HIDDEN" && formValues.Value.trim().length > 0) {
+                    formValues.Value = validationData.updatedVariables[formValues.Name];
+                    if (form) {
+                        form.setValue(formValues);
+                    }
+                    delete that.attributes.BackupVariables;
                 } else {
                     delete that.attributes.BackupVariables;
                 }
```

Comment:
You can replace 
```
[...]
     delete that.attributes.BackupVariables;
 } else {
     delete that.attributes.BackupVariables;
}
```
by
```
[...]
}
delete that.attributes.BackupVariables;
```

Reply:
unfortunately not, because the PR does not show the if case which does not call delete that.attributes.BackupVariables;

Reply:
actually you're right, the if case calls return immediately